src/all/hentaifox/build.gradle

```diff
@@ -3,7 +3,7 @@ ext {
     extClass = '.HentaiFoxFactory'
     themePkg = 'galleryadults'
     baseUrl = 'https://hentaifox.com'
-    overrideVersionCode = 6
+    overrideVersionCode = 7
```

Comment:
You are already bumping the multisrc, so no need to double bump it.

Reply:
Makes sense